GameContent/Shell.cs

```diff
@@ -65,6 +65,13 @@ public struct HomingProperties {
 
         internal bool INTERNAL_doRender = true;
 
+        /// <summary>
+        /// Creates a new <see cref="Shell"/>.
+        /// </summary>
+        /// <param name="position">The position of the created <see cref="Shell"/>.</param>
+        /// <param name="velocity">The velocity of the created <see cref="Shell"/>.</param>
+        /// <param name="ricochets">How many times the newly created <see cref="Shell"/> can ricochet.</param>
+        /// <param name="homing">Whether or not the newly created <see cref="Shell"/> homes in on enemies.</param>
```

Comment:
it's a class that has the properties for homing. it doesnt determine "whether it homes in or not"